pstore/kafka/backoff_test.go

```
@@ -0,0 +1,32 @@
+package kafka
+
+// TODO: If optiopay/kafka accepts pull request, remove this file.
```

Thanks for testing -- this is good to know, and something we'll need to address.

There are two types of down, either the server is failing fast with RST when you try to connect, or it's a black hole where the SYN gets retried a number of times. Which is this case?

Also, can I ask why you perpetually have one down? It seems curious, is this for testing?